src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -25,7 +25,7 @@ const styles = StyleSheet.create({
     fontSize: fontSize(0),
     marginBottom: size(3),
   },
-  quantityText: {
+  unit: {
```

Comment:
This can be reverted to `quantityText`  since it's no more specifically styling the unit!

Reply:
reverted